immersive-web-wg-charter.html

```diff
@@ -373,23 +368,21 @@ <h3 id="w3c-coordination">
           </h3>
           <dl>
             <dt>
-              <a href="https://www.w3.org/community/webvr/">WebVR Community
+              <a href="https://www.w3.org/community/immersive-web/">Immersive Web Community
               Group</a>
             </dt>
             <dd>
-              The WebVR Community Group will provide the WebVR API "v2.0" seed
-              specification to begin the standards process. The WebVR API v1.1
+              The Immersive Web Community Group will provide the WebXR Device API seed specification to begin the standards process. The WebVR API v1.1
```

Comment:
We probably want to indicate that we intend to eventually turn down the WebVR 1.1 API so that perpetual maintenance is not implied?